apps/web/components/team/TeamListItem.tsx

```diff
@@ -175,7 +188,10 @@ export default function TeamListItem(props: Props) {
                           variety="danger"
                           title={t("disband_team")}
                           confirmBtnText={t("confirm_disband_team")}
-                          onConfirm={() => props.onActionSelect("disband")}>
```

Comment:
What's the reason for this change? AFAIK is not adding anything.

Reply:
It was part of the modifications I did for @emrysal's changes request.

Reply:
I think what @emrysal suggested was to remove the need for an extra useState. Which could be solved by just passing the mutation isLoading property as a prop.